reviewrot/__init__.py

```diff
@@ -261,3 +262,24 @@ def represent_none(self, _):
     with open(config_path, 'r') as f:
         config = yaml.safe_load(f)
     return config
+
+
+def remove_wip(results):
+    """
+    Removes WIP reviews from results
+
+    Args:
+        results (list): list of BaseReview instances
+
+    Returns:
+        res (list): list of BaseReview instances with WIP
+                    reviews removed
+    """
+
+    res = []
+    for result in results:
+        match = re.match(r'WIP:?|\[WIP\]', str(result.title))
```

Comment:
Not testing for lowercase? I'm assuming that is still fine.

Reply:
FYI, this is the regex that gitlab is using: https://gitlab.com/gitlab-org/gitlab-ce/blob/master/app/models/merge_request.rb#L304. So it should also match 'WIP\s' and be case insensitive

Reply:
Isn't this too broad a match? Any string that contains `WIP` will match, including acronyms etc.

What about:
* `'^WIP:?|^\[WIP\]` - match the prefix
* `^[^A-Za-z0-9_]*WIP[^A-Za-z0-9_]` -- skips all non-alphanumeric characters until the first WIP and expects non-alphanumeric after
* `^[^A-Za-z0-9]*[Ww][Ii][Pp][^A-Za-z0-9_]` -- like the one above, but also case-insensitive

Reply:
my bad, thanks for the suggestions. I changed the regex to what gitlab is using.